ui/pages/confirmations/components/confirm/info/info.tsx

```diff
@@ -34,6 +35,7 @@ const Info = () => {
       [TransactionType.tokenMethodTransfer]: () => TokenTransferInfo,
       [TransactionType.tokenMethodTransferFrom]: () => NFTTokenTransferInfo,
       [TransactionType.tokenMethodSafeTransferFrom]: () => NFTTokenTransferInfo,
+      [TransactionType.simpleSend]: () => NativeTransferInfo,
```

Comment:
[nit] relocate to under `TransactionType.deployContract` to aim for alphabetizing. We should alphabetize the other items in this list as it is growing